### PR TITLE
Release 7.0.0

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Comment by ekohl** [comment]
Looks like our module is incompatible with puppetlabs-apache's next major version, will look into this. It makes the tests very red.

**Comment by ekohl** [comment]
https://github.com/puppetlabs/puppetlabs-apache/pull/2275

**Comment by ehelms** [comment]
Do we need to pin the version or are you confident we will get a release of puppetlabs-apache before we need to release?

**Comment by ekohl** [comment]
I'm fairly confident it'll be merged soon. And it's just CI where we check out master, in metadata.json we pin to the stable release. The module is preparing a new major version at the end of the week.

I am tempted to get that new major version in Foreman 3.4. They're planning to release it on Friday, so perhaps there's sufficient time. It mostly made things stricter in terms of data types, but also dropped some useless Apache modules that were dead for ages. As long as the manifest compiles, I don't expect problems.

**Comment by ekohl** [comment]
It was merged. Kicking off the tests again.

**Comment by ekohl** [comment]
So this is green, but from a user perspective this module hasn't changed and a major version causes considerable churn. so I'm waiting for more changes.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+[Full Changelog](https://github.com/theforeman/puppet-pulpcore/compare/6.0.0...7.0.0)
+
+**Breaking changes:**
+
+- Drop EL7 support [\#257](https://github.com/theforeman/puppet-pulpcore/pull/257) ([ehelms](https://github.com/ehelms))
+
+**Implemented enhancements:**
+
+- Update to voxpupuli-test 5 [\#258](https://github.com/theforeman/puppet-pulpcore/pull/258) ([ekohl](https://github.com/ekohl))
+- Add Pulpcore 3.18 support [\#256](https://github.com/theforeman/puppet-pulpcore/pull/256) ([ianballou](https://github.com/ianballou))
```
Comment (ehelms) [inline CHANGELOG.md:14]: Is a minor release with this change present warranted at least? As this tends to try and match what Katello requires for its release

Comment (ekohl) [reply CHANGELOG.md:14]: I thought about that (https://github.com/theforeman/puppet-pulpcore/pull/257#pullrequestreview-1042295029) It is only a docs change, but I can do that just for consistency. We have shipped Katello 4.5 with Pulpcore 3.18 before without it though, so technically we're very late with it.

**Review by evgeni — APPROVED** [review]

**Pull request merged**